modules/fluid_properties/doc/content/source/fluidproperties/TemperaturePressureFunctionFluidProperties.md

```diff
@@ -9,15 +9,36 @@ functions. The following relations hold true:
 \begin{aligned}
   x = T \\
   y = P \\
-  \rho = \rho^{user}(time=0, (x=T, y=P, z=0)) \\
-  \mu = \mu^{user}(time=0, (x=T, y=P, z=0)) \\
-  k = k^{user}(time=0, (x=T, y=P, z=0)) \\
+  \rho = \rho^{user}(t=0, (x=T, y=P, z=0)) \\
+  \mu = \mu^{user}(t=0, (x=T, y=P, z=0)) \\
+  k = k^{user}(t=0, (x=T, y=P, z=0)) \\
+\end{aligned}
+\end{equation}
+
+Both the time (`t`) and Z-axis dimension are not used here. A fluid property made to depend on time will
+not be properly updated by this `FluidProperties` object.
+There are two options for specific heat. Either the user sets a constant specific isochoric heat capacity
+
+\begin{equation}
+\begin{aligned}
   c_v = c_v^{user} \\
-  e = c_v * T
+  e = e_{ref} + c_v * (T - T_{ref})
+\end{aligned}
+\end{equation}
+
+Or, the user uses a function of temperature and pressure (same arguments as for density)
+
+\begin{equation}
+\begin{aligned}
+  x = T \\
+  y = P \\
+  cp = cp^{user}(t=0, (x=T, y=P, z=0))
+  cv = cp - \dfrac{\alpha ^ 2 T}{\rho \beta_T}
 \end{aligned}
 \end{equation}
 
-with $T$ the temperature, $P$ the pressure, $\rho$ the density, $\mu$ the dynamic viscosity, $k$ the thermal conductivity, $c_v$ the specific isochoric heat capacity, $e$ the specific energy and the $user$ exponent indicating a user-passed parameter.
+with $T$ the temperature, $P$ the pressure, $\rho$ the density, $\mu$ the dynamic viscosity, $k$ the thermal conductivity, $c_v$ the specific isochoric heat capacity, $e$ the specific internal energy, $T_{ref}$ a reference temperature at which the specific internal energy is equal to a reference energy $e_{ref}$, $\alpha$ the coefficient of thermal expansion, $\beta_T$ the isothermal
+compressibility, and the $user$ exponent indicating a user-passed parameter.
```

Comment:
I recommend some one-letter notation for the superscript. "alpha^user" looks like notation for a user-defined alpha coefficient.